estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,108 @@
+                    <group>
+                        <group>
+                            <group>
+                                <field name="property_type_id" />
+                                <field name="postcode" />
+                                <field name="date_availability" string="Available from" />
+                            </group>
+                            <group>
+                                <field name="expected_price" />
+                                <field name="best_price" />
+                                <field name="selling_price" />
+                            </group>
+                        </group>
+                        <notebook>
+                            <page string="Description">
+                                <group>
+                                    <field name="description" />
+                                    <field name="bedrooms" />
+                                    <field name="living_area" />
+                                    <field name="facades" />
+                                    <field name="garage" />
+                                    <field name="garden" />
+                                    <field name="garden_area" />
+                                    <field name="garden_orientation" />
+                                    <field name="total_area" />
+                                </group>
+                            </page>
+                            <page string="Offers">
+                                <field name="offer_ids" />
+                            </page>
+                            <page string="Other Info">
+                                <group>
+                                    <field name="salesperson_id" />
+                                    <field name="buyer_id" />
+                                </group>
+                            </page>
+                        </notebook>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="estate_property_view_search" model="ir.ui.view">
+        <field name="name">estate.property.form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Properties">
+                <field name="name" string="Title" />
+                <field name="postcode" />
+                <field name="expected_price" />
+                <field name="bedrooms" />
+                <field name="living_area" />
+                <field name="facades" />
+                <separator />
+                <filter string="Available Properties" name="available"
+                    domain="[('state', 'in', ['new', 'offer_received'])]" />
+                <group expand="1" string="Group By">
+                    <filter string="Post Code" name="postcode"
+                        context="{'group_by':'postcode','residual_visible':True}" />
```

Comment:
I don't know where you found that, but it seems unused (and useless?) in recent versions of Odoo
```suggestion
                        context="{'group_by':'postcode'}" />
```